packages/block-editor/src/hooks/block-binding-support/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * WordPress dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+
+/**
+ * Internal dependencies
+ */
+import withBlockBindingSupport from './with-block-binding-support';
+
+export const BLOCK_BINDINGS_ALLOWED_BLOCKS = {
```

Comment:
Is this exposing the `BLOCK_BINDINGS_ALLOWED_BLOCKS` variable in the package? We want to keep it private until we defined a better opt-in mechanism.

Reply:
it's the same thing that [use-binding-attributes does](https://github.com/WordPress/gutenberg/blob/f404337e0cea9511c704323efc088e01eaa11028/packages/block-editor/src/hooks/use-bindings-attributes.js#L25). In in the end it doesn't expose it but anyway let me check, and maybe do not export it because we've introduced the isItPossibleToBindBlock() helper

Reply:
`BLOCK_BINDINGS_ALLOWED_BLOCKS` is not exported anymore.